schema.sql

```diff
@@ -63,4 +63,13 @@ CREATE TABLE visits (
   animal_id INT REFERENCES animals(id),
   vet_id INT REFERENCES vets(id),
   date_of_visit DATE NOT NULL
-)
\ No newline at end of file
+)
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX ix_visit_animal ON visits (animal_id);
+
+CREATE INDEX ix_visits ON visits (vet_id, id, animal_id, date_of_visit);
+
+CREATE INDEX ix_visits ON visits (vet_id, id, animal_id, date_of_visit);
```

Comment:
- [ ] Thank you for adding the screenshots. I can see that the query planner used index for SELECT * FROM owners where email = 'owner_18327@mail.com' and the time decreased.

But for the other two - nothing has changed.

Can you please run again the index creation queries:

```
CREATE INDEX animal_idx ON visits (animal_id);
CREATE INDEX vet_idx ON visits (vet_id);
```

Reply:
Hi @Olamarx ! thank you for reviewving this pr! 🎯 and for pointing this issue out!

My bad 😄 , I have re enter the same Index creator instead of the correct one.

Changes will be made to mitigate this issue.

Cheers!